### PR TITLE
Scroll back to top on menu clicks

### PR DESCRIPTION
**Pull request by anselmbradford**
Always go to the top of the page as navigating between menus.

## Changes

- Scroll back to top on menu changes.
- Turn suspended checks into guard clauses.

## Testing

- `gulp build`
- Open in mobile view.
- Go to About Us, scroll down till Careers button is at the top of the screen.
- Click Careers entries. 
Should scroll to top and slide careers menu into view.

## Review

- @jimmynotjim 
- @KimberlyMunoz 
- @sebworks

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Nice fix :+1:

File: cfgov/unprocessed/js/organisms/MegaMenuMobile.js [hunk 1]
```diff
@@ -74,17 +74,16 @@ function MegaMenuMobile( menus ) {
...
-        _handleTriggerClickBinded( event );
-      } else if ( event.type === 'expandBegin' ) {
-        _handleExpandBeginBinded( event );
-      } else if ( event.type === 'collapseBegin' ) {
-        _handleCollapseBeginBinded( event );
-      } else if ( event.type === 'collapseEnd' ) {
-        _handleCollapseEndBinded( event );
-      }
-    }
+    if ( _suspended ) { return; }
```
Comment (jimmynotjim) [inline cfgov/unprocessed/js/organisms/MegaMenuMobile.js]: Mixed `{}` and not for single line conditions. We should standardize on this, if not for the project in the entire front-end team

Comment (KimberlyMunoz) [inline cfgov/unprocessed/js/organisms/MegaMenuMobile.js]: I'm surprised the linter doesn't have a rule for this.

Comment (anselmbradford) [inline cfgov/unprocessed/js/organisms/MegaMenuMobile.js]: Fixed! We are allowing single line conditionals to not have braces [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/.eslintrc#L540). We could swap that to false, but will need to add braces in a few places.

**Comment by kave** [comment]
👍

**Pull request merged**